listenbrainz/webserver/static/js/jsx/spotify-player.jsx

```diff
@@ -42,6 +42,7 @@ export class SpotifyPlayer extends React.Component {
     this.isCurrentListen = this.isCurrentListen.bind(this);
     this.playListen = this.playListen.bind(this);
     this.playNextTrack = this.playNextTrack.bind(this);
+    this.debouncedPlayNextTrack = _.debounce(this.playNextTrack, 500, {leading:true, trailing:false});
```

Comment:
what is 500 here? half a second?

Reply:
Correct. This being the only place we use the debounced function, you can still click next like  a madman on the controls